### PR TITLE
Clarity edits

### PR DESCRIPTION
**Pull request by eaglepdx**
(1) Worked on Keras Overview; (2) Changed wording and case of title of regression example topic.

### CODE REVIEW

**Comment by lamberta** [comment]
Great! Thank you
Can you please sign the CLA here: https://cla.developers.google.com/ 
After signing, you must add a comment in this pull request with: `I signed it!`
This is a one-time thing.

File: site/en/r2/tutorials/keras/basic_regression.ipynb [hunk 1]
```diff
@@ -75,7 +91,7 @@
         "id": "EIdT9iu_Z4Rb"
       },
       "source": [
-        "# Predict fuel efficiency: regression"
+        "# Regression Example: Predict Fuel Efficiency"
```
Comment (brilee) [inline site/en/r2/tutorials/keras/basic_regression.ipynb]: Predicting

File: site/en/r2/tutorials/keras/basic_regression.ipynb [hunk 2]
```diff
@@ -85,17 +101,17 @@
         "id": "bBIlTPscrIT9"
       },
       "source": [
-        "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
```
Comment (brilee) [inline site/en/r2/tutorials/keras/basic_regression.ipynb]: @lamberta do you know if these escapes are necessary?

**Comment by lamberta** [comment]
Quick ping on this @eaglepdx,
To accept pull requests we need you to sign the CLA here: https://cla.developers.google.com/ 
Then, after signing, you must add a comment in this pull request with: `I signed it!`
Thank you!

**Comment by lamberta** [comment]
Thanks, confirmed CLA signed in https://github.com/tensorflow/docs/pull/592

File: site/en/r2/tutorials/keras/basic_regression.ipynb (hunk 2 above)
Comment (lamberta) [reply site/en/r2/tutorials/keras/basic_regression.ipynb]: The escapes come from saving with internal Colab (vs public Colab). It's annoying

**Pull request merged**